utils/dates.ts

```diff
@@ -0,0 +1,13 @@
+import { formatDistance } from "date-fns";
+
+export const remainingTime = (date: Date): string => {
+  const now: Date = new Date();
+  const timeDiff: number = new Date(date).getTime() - now.getTime();
+  const isInThePast: boolean = timeDiff < 0;
```

Comment:
We can also use `isPast` from `date-fns`

```suggestion
  const isInThePast = isPast(date)
```